_posts/2025-09-10_community-survey-findings.md

```diff
@@ -0,0 +1,178 @@
+  y-axis "Users" 0 --> 26
+  bar [25,17,2,2,1,1,0,0,0,0,0,0,4,5,3,5,5,3,9,0,0,8,4,0,5,1,1,1,1,1,1]
+  line [25,17,2,2,1,1,0,0,0,0,0,0,4,5,3,5,5,3,9,0,0,8,4,0,5,1,1,1,1,1,1]
+
+```
+
+From this we can see that the majority is currently running on Windows (47 users across the various versions), followed by Linux users across a few different distros (57 users
+across Mint(9), Ubuntu(11), Fedora (3), Debian(13), FlatPak(5), Mageia(1), Slackware(2), and Arch(9)).
+
+Now, at the time of the survey we had only released binaries for the Linux Platforms, and we do not have direct support for FlatPak yet (though we are aware that someone in the
+community has at least tried to make a FlatPak build). So it would be natural to expect to see more Linux users at this time. The primary reason for this support was simply
+that we were still working out kinks in the Windows and Mac builds and had not yet developed installers for them as of Vega Strike 0.8.0. Since then we have released
+Vega Strike 0.9.0 with binaries for Windows and Mac; though our Mac builds are presently limited to the Intel platform.
+
+## Platform: Hardware
+
+We also asked about Hardware Architectures:
+
+```mermaid
+xychart-beta horizontal
+    title "Hardware Architecture Usage"
+    x-axis "Processor Architecture" ["Mac M1/M2/M3 ARM", "Intel x86-64", "ARM (RPi, Snapdragon, etc)", "AMD x64"]
+    y-axis "Users" 0 --> 55
+    bar [2, 52, 2, 1]
+    line [2, 52, 2, 1]
+```
+
+We do see some desire for ARM-based releases; though as expected the majority are all x86-64 users.
+
+We are aware that Apple is moving away from Intel and newer Macs are only on their M-series ARM processors.
+We have attempted to support the M-series in our pipeline - GitHub does support it; however, it has been
+problematic for us to support as there have been some build issues and we do not have anyone on the team
+to fully diagnose those right now.
+
+So if you would like to see us support the M-series then it would be greatly helpful if you could join
+our Gitter.im channel. We could really use some folks that can build the project locally, even if we needed
+to help out with getting things setup. Or even if you wanted to provide access to teams members in some form.
+
+Alternatively, the we do have the ability to collect donations through [OpenCollective](https://opencollective.com/vega-strike).
+Our funds there are presently very limited, and our goal is to be able to pay back those in the community that
+are already providing resourcse (DNS, Servers) and cover any storage costs we may need to for our CI system
+(presently supported via the Open Source plan with Docker Hub). However, if there were sufficient funds we
+could also support using services like https://www.macincloud.com/ to provide developers access to the Mac
+Platform to help iron out those issues and deliver better Mac support.
```

Comment:
I have an M1 I can try to build on.

Reply:
that'd be awesome. I know we tried it a while back via GitHub but there were issues; I don't know what all they were any more.